src/cljc/game/core/card.cljc

```diff
@@ -123,6 +123,13 @@
   (= (get-zone card) #?(:clj [:servers :rd :content]
                         :cljs ["servers" "rd" "content"])))
 
+(defn in-remote-root?
+  [card remote]
+  (let [remote-key (if (keyword? remote) remote (keyword remote))
+        remote-str (if (keyword? remote) (name remote) remote)]
+    (= (get-zone card) #?(:clj [:servers remote-key :content]
+                          :cljs ["servers" remote-str "content"]))))
```

Comment:
```suggestion
  (let [remote #?(:clj remote
                  :cljs (if (keyword? remote) (name remote) remote))]
    (= (get-zone card) #?(:clj [:servers remote :content]
                          :cljs ["servers" remote "content"]))))
```

Reply:
There's never any time the server will have a string version of the zone, and it will be rare that we pass a keyword literal in the client.